notes/stories/film/film-scifi-1980s.st.txt

```diff
@@ -5360,6 +5360,7 @@ romantic love [Paul and Chani.]
 uncle and nephew [Baron Harkonnen and Glossu Rabban.]
 mother and daughter [Lade Jessica and Alia.]
 what if I had to fight to the death [Feyd accepted Paul's challenge to a knife fight to the death and lost.]
+force field generator [People surrounded themselves in personal force fields for defense. Most notably, Paul and Duncan had a practice knife fight while enveloped in such protective energy fields.]
```

Comment:
I think the theme should be simply "force field"

Reply:
I couldn't agree more.